runtime/src/main/java/org/corfudb/protocols/CorfuProtocolCommon.java

```diff
@@ -92,9 +94,14 @@ public static UUID getUUID(UuidMsg msg) {
     /**
      * Returns the Protobuf representation of a Layout.
      *
-     * @param layout   the desired (Java) Layout
+     * @param layout   the desired Java Layout object
+     * @return         an equivalent Protobuf Layout message
      */
     public static LayoutMsg getLayoutMsg(Layout layout) {
+        if (layout == null) {
+            return LayoutMsg.getDefaultInstance();
+        }
+
```

Comment:
This method is not be used.

Reply:
You can see it used in PR 2823. We included it in common for easier availability since it will be required by both the management and layout RPCs.